lnd.go

```diff
@@ -268,6 +268,9 @@ func Main(cfg *Config, lisCfg ListenerCfg, implCfg *ImplementationCfg,
 
 	rpcServerOpts := interceptorChain.CreateServerOpts()
 	serverOpts = append(serverOpts, rpcServerOpts...)
+	serverOpts = append(
+		serverOpts, grpc.MaxRecvMsgSize(lnrpc.MaxGrpcMsgSize),
```

Comment:
Were we previously not setting this for the server?

Reply:
I believe we missed this before and so it was defaulting to 4 MiB.